### PR TITLE
188 setuptools

### PR DESCRIPTION
**Pull request by mhubii**
Addresses #188 

# Description
- Introduces setuptools
- Removes relative paths from examples
- Updates readme

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+    "scikit-image",
+    "scipy"
+]
+
+if __name__ == '__main__':
+    setup(
+        # Metadata
+        name="monai",
+        version=VERSION,
+        author="MONAI Core Team",
```
Comment (wyli) [inline setup.py]: author could be "MONAI Consortium"

File: setup.py [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+        description="AI Toolkit for Healthcare Imaging",
+        long_description_content_type="text/markdown",
+        long_description=readme,
+        license="Apache License 2.0",
+
+        # Package info
+        packages=find_packages(exclude=('docs', 'examples', 'tests')),
+        zip_safe=True,
+        install_requires=requirements
+    )
```
Comment (wyli) [inline setup.py]: flake8 asks for a newline at the end of this file
```
flake8 . --count --statistics --config .flake8
./setup.py:55:6: W292 no newline at end of file
1     W292 no newline at end of file
1
```

File: setup.py [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+]
+
+if __name__ == '__main__':
+    setup(
+        # Metadata
+        name="monai",
+        version=VERSION,
+        author="MONAI Core Team",
+        url="https://github.com/Project-MONAI/MONAI",
+        author_email="wenqil@nvidia.com",
```
Comment (wyli) [inline setup.py]: email could be monai.miccai2019@gmail.com

File: setup.py (hunk 2 above)
Comment (mhubii) [reply setup.py]: its good practice to put a new line at the end of a file, hence the warning

File: setup.py (hunk 3 above)
Comment (mhubii) [reply setup.py]: changed mail

File: README.md [hunk 4]
```diff
@@ -19,6 +19,16 @@ MONAI is a [PyTorch](https://pytorch.org/)-based, [open-source](https://github.c
 - customizable design for varying user expertise;
 - multi-GPU data parallelism support.
 
+## Installation
+Clone and build this repository from source
+
+```bash
+git clone https://github.com/Project-MONAI/MONAI.git
+python setup.py install
```
Comment (wyli) [inline README.md]: forgot to check this just now, i think we need `cd MONAI/` before python setup.py, or `pip install -e MONAI/` works fine as well

Comment (mhubii) [reply README.md]: oh yes you are right

**Pull request merged**